SerialSample/CPP/Generated Files/XamlTypeInfo.Impl.g.cpp

```diff
@@ -95,6 +95,41 @@ ::Windows::UI::Xaml::Markup::IXamlMember^ ::XamlTypeInfo::InfoProvider::XamlType
 }
 
 
+::Windows::UI::Xaml::Markup::IXamlType^ ::XamlTypeInfo::InfoProvider::XamlTypeInfoProvider::CheckOtherMetadataProvidersForName(::Platform::String^ typeName)
```

Comment:
I tried to check-in the generated files that were already checked in. I did not want to remove any that were already there but I did not add any new ones